packages/ruleset/test/error-response-schemas.test.js

```diff
@@ -78,7 +78,7 @@ describe(`Spectral rule: ${ruleId}`, () => {
         `Error container model should contain property 'trace' of type string`
       );
       expect(r.path.join('.')).toBe(
-        'paths./v1/drinks.post.responses.400.content.application/json.schema.properties'
+        'paths./v1/cars.post.responses.400.content.application/json.schema.properties'
```

Comment:
These are interesting - looks like it's not just the order changed but the path caught is different for every change in this test?

Reply:
these are situations where multiple errors are returned but we're only validating the first element in the results array, and the re-ordering means a completely different path is associated with the new first error element.